qutebrowser/javascript/webelem.js

```diff
@@ -202,68 +211,99 @@ window._qutebrowser.webelem = (function() {
         return true;
     }
 
-    // Returns true if the iframe is accessible without
-    // cross domain errors, else false.
-    function iframe_same_domain(frame) {
-        try {
-            frame.document; // eslint-disable-line no-unused-expressions
-            return true;
-        } catch (err) {
-            return false;
+    // Runs a function in a frame until the result is not null, then return
+    // If no frame succeds, return null
+    function run_frames(func) {
+        for (let i = 0; i < window.frames.length; ++i) {
+            const frame = window.frames[i];
+            if (utils.iframe_same_domain(frame)) {
+                const result = func(frame);
+                if (result) {
+                    return result;
+                }
+            }
         }
+        return null;
+    }
+
+
+    // Return elements that are scrollable
+    function find_scrollable(only_visible) {
+        const scrollable_elts =
+              Array.from(document.querySelectorAll("div,ol,ul")).
+                  filter((elt) => elt.clientHeight < elt.scrollHeight &&
+                          scroll.is_scrollable(elt, 0, 1)).
```

Comment:
Might be worthwhile to include horizontal scrolling as well, but I'm not sure (might add clutter for little gain)